charts/incubator/kubernetes-reflector/templates/common.yaml

```
@@ -0,0 +1,5 @@
+{{/* Make sure all variables are set properly */}}
+{{- include "tc.v1.common.loader.init" . }}
+
+{{/* Render the templates */}}
+{{ include "tc.v1.common.loader.apply" . }}
```

```suggestion
{{- include "tc.v1.common.loader.all" . }}
```
Those can be combined